docs/app/docs/devbox_cloud/browser_getting_started.md

```diff
@@ -14,7 +14,7 @@ If you are trying to open a private repository in Devbox Cloud, you will first h
 :::
 
 1. Navigate to the Github Repo that you want to preview in Devbox Cloud
-2. Copy the URL, and prepend it with `https://devbox.sh/`. For example, to open the Devbox repo, you would enter [https://devbox.sh/github.com/jetpack-io/devbox](http://devbox.sh/github.com/jetpack-io/devbox)
+2. Copy the URL, and prepend it with `https://devbox.sh/`. For example, to open the Devbox repo, you would enter [https://devbox.sh/github.com/jetpack-io/devbox](http://devbox.sh/github.com/jetpack-io/devbox). You can also enter the URL on [devbox.sh](https://devbox.sh) and click the `Open` button
```

Comment:
Click the `Go` button (or should we change the button to `Open`?)

Reply:
I'll edit it to "Go"

Reply:
Not changed @Lagoja ?